webapp/tsconfig.json

```diff
@@ -11,6 +11,13 @@
     "resolveJsonModule": true,
     "strictNullChecks": true,
     "strictBindCallApply": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
Why is the `skipLibCheck` needed?

Reply:
It's to avoid checking the types in `node_modules` and only check the types in the `src` folder. If it is set to `false` I get the following:

```
$ npm run check-types

> pexip-video-connect@1.0.0 check-types
> tsc --noEmit

node_modules/@pexip/components/dist/types/src/components/elements/Audio/Audio.d.ts:2:18 - error TS2430: Interface 'AudioElement' incorrectly extends interface 'HTMLAudioElement'.
  The types returned by 'setSinkId(...)' are incompatible between these types.
    Type 'void' is not assignable to type 'Promise<void>'.

2 export interface AudioElement extends HTMLAudioElement {
                   ~~~~~~~~~~~~

node_modules/@pexip/components/dist/types/src/components/elements/Video/Video.d.ts:5:18 - error TS2430: Interface 'VideoElement' incorrectly extends interface 'HTMLVideoElement'.
  The types returned by 'setSinkId(...)' are incompatible between these types.
    Type 'void' is not assignable to type 'Promise<void>'.

5 export interface VideoElement extends HTMLVideoElement {
                   ~~~~~~~~~~~~

node_modules/@pexip/media-processor/dist/main/video/videoStreamTrackProcessor.d.ts:2:14 - error TS2552: Cannot find name 'MediaStreamVideoTrack'. Did you mean 'MediaStreamTrack'?

2 type Track = MediaStreamVideoTrack | MediaStreamVideoTrackGenerator;
               ~~~~~~~~~~~~~~~~~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:15470:13
    15470 declare var MediaStreamTrack: {
                      ~~~~~~~~~~~~~~~~
    'MediaStreamTrack' is declared here.

node_modules/@pexip/media-processor/dist/main/video/videoStreamTrackProcessor.d.ts:2:38 - error TS2304: Cannot find name 'MediaStreamVideoTrackGenerator'.

2 type Track = MediaStreamVideoTrack | MediaStreamVideoTrackGenerator;
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@pexip/media-processor/dist/main/video/videoStreamTrackProcessor.d.ts:6:41 - error TS2552: Cannot find name 'MediaStreamVideoTrack'. Did you mean 'MediaStreamTrack'?

6 export type ProcessVideoTrack = (track: MediaStreamVideoTrack, transformers: Array<Transformer<InputFrame, InputFrame>>, options?: Options) => Promise<Track>;
                                          ~~~~~~~~~~~~~~~~~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:15470:13
    15470 declare var MediaStreamTrack: {
                      ~~~~~~~~~~~~~~~~
    'MediaStreamTrack' is declared here.


Found 5 errors in 3 files.

Errors  Files
     1  node_modules/@pexip/components/dist/types/src/components/elements/Audio/Audio.d.ts:2
     1  node_modules/@pexip/components/dist/types/src/components/elements/Video/Video.d.ts:5
     3  node_modules/@pexip/media-processor/dist/main/video/videoStreamTrackProcessor.d.ts:2
     ```

Reply:
Yeah I wonder if a proper fix should be done in `@pexip/media-processor` / `@pexip/components`. Any ideas @skyuplam?